### ISSUE TITLE
this plugin doing brrr

### ISSUE THREAD

**Issue by RustemB** [andweeb/presence.nvim#28]
https://www.reddit.com/r/neovim/comments/on6n9t/whats_wrong_i_got_this_every_time_for_example/

**Comment by andweeb** [comment]
Oof when does this happen?

**Comment by RustemB** [comment]
sadly, but i removed this plugin :(
it appears when i open new window, like `:help`, `packer` or `telescope`

**Comment by andweeb** [comment]
Ah no worries, that's understandable :/

It would really help to get some more info to resolve this for other folks that may also be running into the error, since I'm having some trouble replicating this locally.

Would you mind telling me your OS and Neovim version? Has this always been occurring or only after updating the plugin at some point?

**Comment by RustemB** [comment]
OS: GNU/Linux (ArchLinux)
Kernel Info: `Linux machine 5.12.15-zen1-1-zen #1 ZEN SMP PREEMPT Wed, 07 Jul 2021 23:35:33 +0000 x86_64 GNU/Linux`
Neovim version: `NVIM v0.6.0-dev+59-g2475161de` [AUR](https://aur.archlinux.org/packages/neovim-git)

**Comment by andweeb** [comment]
Hmm unfortunately I'm still not able to see the error this despite using the same Neovim version with Telescope on Arch Linux:
<img width="667" alt="Screen Shot 2021-07-21 at 9 16 18 PM" src="https://user-images.githubusercontent.com/10726590/126589264-69e32569-4587-415d-8d23-794764ea2996.png">

The only other relevant difference I can think of is the version of presence.nvim you were using at the time of seeing the error, were you using an outdated version of the plugin?

**Comment by andweeb** [comment]
Regarding the error in particular, it would occur whenever a Lua object is incorrectly passed into the logger. For example running this:
```
:lua package.loaded.presence.log:debug({ hello = true })
```
would yield the error you've noticed:
```
Error executing vim.schedule lua callback: Vim:E731: using Dictionary as a String
```

I sifted through every log in the repo but it doesn't seem like there's any case of this currently, but _just_ to be safe I've just pushed a commit to guard against any potential case of that happening.

Thanks for reporting :) And sorry for the annoyance it may have caused!
